chirp/inference/baw_utils.py

```diff
@@ -0,0 +1,131 @@
+# coding=utf-8
+# Copyright 2024 The Perch Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions for working with a Bioacoustic Workbench (baw) (e.g. A2O) API."""
+
+import io
+import os
+import re
+from typing import Generator, Sequence
+import urllib
+
+from chirp import audio_utils
+import librosa
+from ml_collections import config_dict
+import numpy as np
+import requests
+import soundfile
+
+
+def make_baw_audio_url_from_file_id(
+    file_id: str, offset_s: float, window_size_s: float, baw_domain: str = "data.acousticsobervatory.org"
```

Comment:
There's a typo here, which threw me for a loop: data.acousticsobervatory.org

Is there a reason to prefer the `data` subdomain vs the `api` subdomain we were using before?

Reply:
ah no, my mistake. I will fix